SPRINT-STRUCTURE.md

```diff
@@ -46,8 +46,8 @@ This layout structures the playbook content to make it easy day to day to find r
   - [Feature Branching (creating branch for new story)](source-control/feature-branching/readme.md)
 - Day 2
   - [Source Control](source-control/readme.md)
-    - Commit best practices
-    - [Git guide](source-control/git.md)
+    - [Commit best practices](source-control/contributing/readme.md#commit-best-practices)
```

Comment:
This document no longer looks like this

Reply:
@TessFerrandez please re-review this file.